app/views/users/_form.html.erb

```diff
@@ -0,0 +1,34 @@
+<%# <form action="/users" method="POST" accept-charset="utf-8">
```

Comment:
- Remove commented code.

Reply:
Hey Carlos,

Thank you for your consideration and for taking out time to check my code.
As per your request, I have removed the commented code, but if you check the project requirements they actually want us to leave it there so that we know how to create forms with HTML and form_tag in rails.
[Odin Project](https://www.theodinproject.com/paths/full-stack-ruby-on-rails/courses/ruby-on-rails/lessons/forms) You can see the specifications here.

Cheers,
Happy Coding!!

Reply:
Ok 🙌🏽, if your project, it's done. Request another code review from your dashboard.